mailipy/send.py

```diff
@@ -95,7 +97,9 @@ def send_emails(server, emails, outbox_dir, sent_dir):
 
             # On success, move the message from the outbox to the sent folder
             shutil.move(os.path.join(outbox_dir, eml), os.path.join(sent_dir, eml))
-        except:
+            time.sleep(int(args.sleep))
+
+    except:
```

Comment:
This looks broken, the `except` is now at the same indentation level as `for`, instead of `try`.

(This is one of the reasons I dislike Python by the way 😅 )

Reply:
good idea sleep after exception!

I want control the speed of send, to avoid server load or Reduce the risk of rejection by the counterparty server

Reply:
Yes I understood that. But the code is broken, I get this error:

```
Traceback (most recent call last):
  [...]
  File "mailipy/send.py", line 115
    except:
IndentationError: unexpected unindent
```

Can you fix it?

Reply:
It should work if you make changes from https://github.com/HUI2125281/mailipy/tree/patch-2 directly.

(Don't worry if it says that the branch is a few commits behind, I can merge it later.)